eslint-rules/tests/lib/rules/no-hard-coded-font.js

```diff
@@ -13,35 +13,41 @@ RuleTester.setDefaultConfig({
 
 const ruleTester = new RuleTester();
 
-// ruleTester.run('no-hard-coded-font', rule, {
-//   valid: [
-//     { code: 'const goodUsage = <Text style={{color: \'red\'}}/>;' },
-//     { code: 'const goodUsage = <Text style={{fontSize: Typography.text10.fontSize}}/>;' },
-//   ],
-//   invalid: [
-//     {
-//       code: 'const badUsage = <Text style={{fontSize: 15}}/>;',
-//       errors: [
-//         {
-//           messageId: 'foo',
-//         },
-//       ],
-//     },
-//     {
-//       code: 'const badUsage = <Text style={{fontSize: Constants.isAndroid ? 16 : 17}}/>;',
-//       errors: [
-//         {
-//           messageId: 'foo',
-//         },
-//       ],
-//     },
-//     {
-//       code: 'const badUsage = <Text style={{fontSize: size}}/>;',
-//       errors: [
-//         {
-//           messageId: 'foo',
-//         },
-//       ],
-//     },
-//   ],
-// });
+ruleTester.run('no-hard-coded-font', rule, {
```

Comment:
can you add a good scenario for when some spread the typography preset

Reply:
I added another valid case